integration/spark/app/src/test/java/io/openlineage/spark/agent/GoogleCloudIntegrationTest.java

```diff
@@ -56,8 +58,8 @@ class GoogleCloudIntegrationTest {
   private static final String NAMESPACE = "google-cloud-namespace";
   private static final int MOCKSERVER_PORT = 3000;
   private static final String LOCAL_IP = "127.0.0.1";
-  private static final String SPARK_3 = "(3.*)";
-  private static final String SPARK_3_3 = "(3\\.[3-9].*)";
+  private static final String GREATER_THAN_SPARK2 = "(3.*)";
+  private static final String GREATER_THAN_SPARK3_3 = "(3\\.[3-9].*)";
```

Comment:
Should those both contain Spark 4? If not, why?

Reply:
As mentioned few lines above:
```
// TODO: Please note the test remains disabled for Spark 4.0 for now (no applicable connector
// version available)
```